File: ClientApp/src/app/shared/models/token-model.ts
export class TokenModel {
    cif: string;
    name: string;
    isAdmin: boolean;
    bankAccount: string;
    token: string;
    validTo: Date;

    constructor(result: TokenModel)
    {
        this.cif = result.cif;
        this.name = result.name;
        this.isAdmin = result.isAdmin;
        this.bankAccount = result.bankAccount;
        this.token = result.token;
        this.validTo = result.validTo;
    }
}

File: Model/TokenModel.cs
using System;
using System.Collections.Generic;

namespace coop2._0.Model
{
    public class TokenModel
    {
        public string Cif { get; set; }
        public string Name { get; set; }
        public bool IsAdmin { get; set; }
        public int BankAccount { get; set; }
        public string Token { get; set; }
        public DateTime ValidTo { get; set; }
    }
}
